app/controllers/user_controller.rb

```diff
@@ -0,0 +1,14 @@
+# Controller for user features not provided by Devise - profile pages, etc
+class UserController < ApplicationController
```

Comment:
Consider renaming to `UsersController` - the pluralized form is more conventional and plays better with some Rails features, in particular `resource` and `resources` routes, e.g. in `config/routes.rb`:

```rb
# Singular resources
resource :user # assumes controller is named UsersController

# Plural resources
resources :users # also assumes controller is named UsersController
```

https://guides.rubyonrails.org/routing.html#singular-resources